windowing_library.py

```diff
@@ -0,0 +1,65 @@
+import pickle
+import numpy as np
+from skimage.util.shape import view_as_windows
+import statistics
+
+def create_windows(matrix, rows, window_size, stride, mod_num):
+  x = matrix['x']
+  y = matrix['y']
+  z = matrix['z']
+  nanos = matrix['nanos'] 
+  np_data = np.array([x,y,z,nanos], 'd')
+
+  windowed_data = view_as_windows(np_data, (rows, window_size), step=stride)
+  return windowed_data
+
+def find_initial_lift_times(lift_windows, rows):
+  start = lift_windows[0][rows-1][0]
+  end = lift_windows[-1][-1][-1]
+  return start, end
+
+def find_lift_windows(windowed_data, threshold_divisor):
```

Comment:
What's the purpose of the threshold_divisor? This might all become clearer with function comments :-)